packages/pigeon/platform_tests/ios_unit_tests/ios/Runner/async_handlers.h

```diff
@@ -0,0 +1,27 @@
+// Autogenerated from Pigeon (v0.1.19), do not edit directly.
```

Comment:
I am just going to comment on the generated files.  Most of them aren't related to this specific change, just the generated files as a whole, so take as you will.